.github/workflows/create-release-notes.yaml

```
@@ -0,0 +1,26 @@
+name: Generate Release Notes
+
+on:
+  push:
+    tags:
+      - "*"  # Triggers on any tag creation
+
+jobs:
+  generate-release-notes:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
```

@v3 are obsolete versions, lets use latest @v4
```suggestion
      - name: Checkout repository
        uses: actions/checkout@v4

      - name: Set up Node.js
        uses: actions/setup-node@v4
```